style.css

```diff
@@ -355,3 +356,151 @@ body {
 .arrow1 {
   margin-left: 205px;
 }
+
+.contact {
+  padding-top: 80px;
+  position: relative;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: #6070ff;
+  background-image: url(./img/contact\ form\ shapes-mobile.svg);
+  background-repeat: no-repeat;
+  background-position: 130% 20%;
+  border-radius: 75px 0 0 0;
+}
+
+.contact_footer p {
+  width: 327px;
+  height: 102px;
+  justify-content: center;
+  font-family: "poppins", "Trebuchet MS", "Lucida Sans Unicode", sans-serif;
+  font-style: normal;
+  font-weight: 400px;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+.contact h2 {
+  padding-left: 47px;
+  align-items: center;
+  font-family: 'poppins', "Trebuchet MS", "Lucida Sans Unicode", sans-serif;
+  font-style: normal;
+  font-weight: 700px;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  row-gap: 20px;
+  margin: 70px 0;
+  font-family: 'Poppins', sans-serif;
+  align-items: start;
+  width: 327px;
+}
+
+form input {
+  padding: 10px;
+  height: 40px;
+  width: 100%;
+}
+
+form textarea {
+  padding: 10px;
+  width: 100%;
+  outline: none;
+  font-family: 'Poppins', sans-serif;
+  height: 170px;
+  top: 5140px;
+  left: 21px;
+}
+
+.start {
+  width: 150px;
+  font-weight: 500;
+  margin-bottom: 40px;
+  padding: 12px 16px;
+  height: 48px;
+  left: 24px;
+  top: 5317px;
+  background: #fff;
+  border-radius: 8px;
+  color: #6070ff;
+  font-size: 17px;
+  border-color: #f2f5fc;
+  transition: color;
+  transition-duration: 1s;
+  transition-delay: 0.1s;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+}
+
+.start:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.start:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+/*
+form {
+  display: flex;
+  flex-direction: column;
+  row-gap: 20px;
+  margin: 70px 0;
+  font-family: 'Poppins', sans-serif;
+  align-items: start;
+  width: 327px;
+}
+
+form input {
+  padding: 10px;
+  height: 40px;
+  width: 100%;
+  border-radius: 8px;
+  font-size: 17px;
+  font-weight: 400;
+}
+
+form textarea {
+  padding: 10px;
+  width: 100%;
+  outline: none;
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  height: 170px;
+  top: 5140px;
+  left: 21px;
+  border-radius: 8px;
+}
+
+.start {
+  width: 150px;
```

Comment:
[OPTIONAL]
- [ ] Please don't leave commented code in your code base, it makes your code unclean.